actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BaseBulkUploadBackgroundJobActor.java

```diff
@@ -83,10 +85,18 @@ public void processBulkUpload(BulkUploadProcess bulkUploadProcess, Function func
       queryMap.put(BulkUploadJsonKey.SEQUENCE_ID, sequenceRange);
       List<BulkUploadProcessTask> tasks = bulkUploadProcessTaskDao.readByPrimaryKeys(queryMap);
       function.apply(tasks);
+      for (BulkUploadProcessTask task : tasks) {
+        if (task.getStatus() == ProjectUtil.BulkProcessStatus.FAILED.getValue()) {
```

Comment:
Should be as follows: this is capital integer - can't be compared with double equals
ProjectUtil.BulkProcessStatus.FAILED.getValue().equals(task.getStatus())